TWLight/users/templates/users/preferences.html

```diff
@@ -28,14 +31,15 @@ <h3>Preferences</h3>
   {% if user.has_usable_password %}
     <div class="row clearfix">
       <div class="col-xs-12 col-sm-3">
-        {% comment %} Translators: This is the heading of the section of a user's profile where they can change their password. {% endcomment %}
+        {% comment %} Translators: This is the heading of the section of a user's profile where users can change their password. {% endcomment %}
         <strong>{% trans "Password" %}</strong>
       </div>
-      <div class="col-xs-12 col-sm-9">
+      <div class="col-xs-12 col-sm-4">
         <form method="post" action="{% url 'password_change' %}">
           {% csrf_token %}
           {{ password_form | crispy }}    
           <button class="btn btn-default" type="submit">
+            {% comment %} Translators: In the preferences section of a user profile, this text labels the button users can click to change their password. {% endcomment %}
             {% trans "Change password" %}
```

Comment:
We should unset this section as translatable. No regular editor accounts are going to see the password change section.
Also, I'm not sure that css changes belong in this pr, but I won't insist that you take the change out.